tests/unit/backend/modules/verification/verification.service.spec.ts

```diff
@@ -280,25 +280,25 @@ describe('Verification service', () => {
       expect(foundTransaction.fields[0].hashRetries).toBe(0)
     })
 
-    test('should send OTP for mobile field', async () => {
-      // Reset field so we can test update later on
+    it('should send OTP when params are valid for mobile field', async () => {
+      // Reset field so we can it update later on
```

Comment:
😆 
```suggestion
      // Reset field so we can test update later on
```

Reply:
computers are dumb

Reply:
damn I'm gonna regret saying that during the robot apocalypse